vara/transport.go

```diff
@@ -40,6 +55,25 @@ func (m *Modem) DialURL(url *transport.URL) (net.Conn, error) {
 		return nil, err
 	}
 
+	// Listen on
+	if err := m.writeCmd(fmt.Sprintf("LISTEN ON")); err != nil {
+		return nil, err
+	}
+
+	if m.scheme == "varahf" {
+		// VaraHF only - Winlink or P2P?
+		p2p := url.Params.Get("p2p") == "true"
```

Comment:
PAT does not, however Vara does, at least according to the manual in the repo https://github.com/n8jja/Pat-Vara/blob/main/VARA%20Protocol%20Native%20TNC%20Commands.pdf

Since Vara is closed source I could not do much besides sniffing the traffic between RMS Express and Vara, comparing it to the manual and using the same.